src/page-building/service/industries.tsx

```diff
@@ -0,0 +1,53 @@
+import { Teaser } from '../../components/teasers/teaser';
+import { Intro } from './support';
+import React from 'react';
+import { ContentBlockContainer } from '../../components/layout/content-block-container';
+import styled from 'styled-components';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+
+interface IndustriesProps {
+  className?: string;
+}
+
+const TeaserContainer = styled.div`
+  display: grid;
+  gap: 24px;
+  grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
+  margin-top: 40px;
+`;
+
+const TeaserHighlighted = styled(Teaser)`
+  grid-column: 1/-1;
+`;
```

Comment:
I needed to adjust and try a few things, that's why I define that stuff locally for now.